record_tx.go

```diff
@@ -88,27 +88,24 @@ func waitForRecordTxWriteLock(ctx context.Context, c ClientInterface, key string
 	// Relevant for bux to bux transactions, as we have 1 tx but need to record 2 txs - outgoing and incoming
 
 	lockKey := fmt.Sprintf(lockKeyRecordTx, key)
-	c.Logger().Info(ctx, lockKey)
+	c.Logger().Info().Msgf(lockKey)
```

Comment:
Please remove this line - my mistake I didn't do it before

Reply:
Done